lib/perl/Genome/Model/Command/Admin/ModelSummary.pm

```diff
@@ -202,6 +184,21 @@ sub execute {
     return 1;
 }
 
+sub _build_and_status_for_model {
```

Comment:
The first half of this method looks very similar to `current_build` on `Model.pm`, except that `current_build` also checks if the inputs are up to date. I imagine doing that would slow this down way too much to use in bulk processing? `status_with_build` in `Model.pm` also seems to have some overlap with this method.